playground/src/Playground.tsx

```diff
@@ -49,11 +49,12 @@ export const Playground = ({ code, scope, noInline }: PlaygroundProps) => {
       noInline={noInline}
       theme={oceanicNext}
       transformCode={(code) => {
-        if (code.trim().startsWith('<'))
-          return `<>${code}</>`;
+        if (code.trim().startsWith('<')) {
+          return `<>${code.trim()}</>`;
+        }
 
-        const [firstCharacter = ''] = `${code}`.trim().match(firstCharacterNotCommentRegex) || [];
-        return firstCharacter.startsWith('<') ? `<>${code}</>` : code;
+        const [firstCharacter = ''] = code.trim().match(firstCharacterNotCommentRegex) || [];
+        return firstCharacter.startsWith('<') ? `<> ${code.trim().match(firstCharacterNotCommentRegex)} </>` : code;
```

Comment:
You should extract a const for the result of `code.trim().match(firstCharacterNotCommentRegex)` rather than call it twice.
Also my understanding is that this strips the comments.
What is the actual behaviour? Do you have a url with a preview to this?

Reply:
![image](https://user-images.githubusercontent.com/7051248/115238279-2ebe8880-a126-11eb-9e63-6a63ae809c94.png)
it seems that when there's a comment line, it adds commas between the components
This needs to be investigated

Reply:
In the end, I transformed the code by 1, removing the multiline comments block and 2, display the lines that are not single line comments.

Sample: https://design-systems.dev/edit/wzPn6NO54tDJlBxCtjGx/playground/doc/playground.mdx?branch=~cornelurian@optional_noInline
![image](https://user-images.githubusercontent.com/18328465/115303162-f2158000-a16b-11eb-8871-280fabf13dfa.png)